modules/test/playwright/pages/notification-web/EmailNotificationTemplatePage.ts

```diff
@@ -37,6 +40,8 @@ export class EmailNotificationTemplatePage {
 		this.backURLButton = page.getByTitle('Back', {exact: true}).first();
 		this.basicInfoName = page.getByLabel('Name' + 'Mandatory').first();
 		this.contentSubject = page.getByLabel('Subject' + 'Mandatory');
+		this.copyButton = page.getByRole('button', {name: 'Copy'});
+		this.entity = page.getByText('Select an Option').last();
```

Comment:
is this last() really necessary? there is more than one `Select an Option` selector there?

Reply:
yes, i would suggest to change on the main code for the label to be related to the input or add an `aria-label` for this as well

Reply:
I added an `id` to the field and `fieldBase` was responsible for linking the label to the input using this id.

Reply:
great! this is indeed the way we are usually doing